### ISSUE TITLE
Winmerge stuck when comparing folders in Windows 10

### ISSUE THREAD

**Issue by snowjim** [WinMerge/winmerge#759]
[This problem still exists](https://github.com/WinMerge/winmerge/issues/18), have tried several big folders with subfolders and after a while it gets stuck, no error and no more progress. I have to micromanage the compare and choose just a couple of folders at a time. Its a big risk that I might miss some compares.

I have a failing harddrive so I have copied all content to another disk. To get a list of files that did not make it I need to make a compare. The last time it stoped on item 42497 that is a regular jpg file.

**Comment by sdottaka** [comment]
Is my understanding correct that you were comparing a folder on a failed hard disk with a folder on a good disk?

To test, I need to have a failed hard disk or simulate a hard disk failure, which seems difficult at this time.

**Comment by snowjim** [comment]
Yes, this one might be hard to track down. I thought that you could check the code and see where it might freeze and maybe to some handling but I get that this might not be fixable. Thanks anyway.

**Comment by Tom-W-603** [comment]
I am having a similar problem, Winmerge 2.16.36.0 x64, recently installed, is stuck on an item during a compare and not responding to stop.  It has been on the same item for about 24 hours now.  No failed hard drive in this case, at least as far as I know.  Total Items 858958, stuck on item 506569.  I suspect the issue may be the folder name: Sallie (Werst) Havens on Instagram_ “We dance.... #dance #dream #live #love #happyplace #justbeyou #multiplepersonalities #choosejoy 📸 @timothygavery”_files.  Any suggestions on how to get past this?

Thanks,
Tom
![image](https://github.com/WinMerge/winmerge/assets/155384933/ccd0544f-07f2-487e-9a88-0aa19a3c2ba7)
![Winmerge Stuck](https://github.com/WinMerge/winmerge/assets/155384933/e187b1a9-a2c3-47f3-94c6-ae5f30f6b0a8)

**Comment by sdottaka** [comment]
I tried to reproduce this issue as below but couldn't do it.
It would be helpful if you could tell me more detailed reproduction conditions.
![image](https://github.com/WinMerge/winmerge/assets/98126/3487b67b-7f91-4c64-a1e2-acbd9bc0e0b4)
